src/services/doppler-mvc-client.ts

```diff
@@ -0,0 +1,124 @@
+import { AxiosInstance, AxiosStatic } from 'axios';
+import { Color } from 'csstype';
+import headerDataJson from '../headerData.json';
+
+// TODO: Consider better the name, DopplerMvc tries to make reference to Doppler API implemented
+// inside MVC App
+
+interface NavEntry {
```

Comment:
it's ok all interfaces in the same file?

Reply:
Mmm, I only them here, what do you think?